app/components/ForumBox/ForumBox.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {View, Text, TextInput, StyleSheet, TouchableOpacity} from 'react-native';
+import styles from './styles';
+
+export default class ForumBox extends React.Component {
+    render() {
+    return (
+        <View style={{marginTop: 15}}>
+                
+                <View style = {styles.QuestionStyle}>
+                    <View style={styles.nameHeadlineStyle}>
+                        <Text style={{marginLeft: 10}}>{this.props.Question}</Text>
+                    </View>
+                    <View style={{backgroundColor:'white', borderRadius:10}}>
+                        <TextInput style={{padding: 10, fontSize: 13, width: "90%", height: 80 }} multiline  placeholder = "Ask your query, by addressing your problem clearly!" />
+                        
+                    </View>
```

Comment:
Remove additional spaces. They make no sense here.

Reply:
Or just use a formatter for code formatting, That would also help you to remove and organize your code more readable manner

Reply:
@shehand Please review